Resources/Prototypes/Entities/Clothing/Eyes/base_clothingeyes.yml

```
@@ -22,3 +22,15 @@
     reflects:
       - Energy
       - NonEnergy
+
+- type: entity
+  abstract: true
+  parent: [Clothing, BaseFoldable]
+  id: FlippableEyesBase
+  components:
+    - type: Foldable
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)